lib/gcloud/bigquery/data.rb

```diff
@@ -58,7 +59,7 @@ def token
 
       # The total number of rows in the complete table.
       def total
-        @gapi.total_rows
+        Service.i_from @gapi.total_rows
```

Comment:
Assuming you want the same solution for Time conversions, will do. Update on the way.